samples/MerQure.Tools.Samples/RetryBusExample/Domain/SampleException.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MerQure.Tools.Samples.RetryBusExample.Domain
+{
+    public class SampleException : Exception
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Update this implementation of 'ISerializable' to conform to the recommended serialization pattern. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS3925)